### ISSUE TITLE
Firebase Cloud Messaging not sending push notifications to Standalone Apple Watch

### ISSUE THREAD

**Issue by sayan-futurehook** [firebase/firebase-ios-sdk#10575]
### Description

I am currently building a watchOS app where I have added FCM. I have added watchOS app bundle identifier and made an iOS app from firebase, and added GoogleService-Info.plist to the watchOS target. I have added AppDelegate code in the watchOS target for registering APN with FCM. When I am trying to send notification using the fcmToken, The watch is not getting notification. After running the app on watch I have accepted the permission to send notification. I have used this example: https://github.com/firebase/firebase-ios-sdk/tree/master/FirebaseMessaging/Apps/SampleStandaloneWatchApp

Code: 
![undefined - Imgur (1)](https://user-images.githubusercontent.com/110037874/207568224-daf87853-1f34-40dc-af97-b85eaff4d7b3.png)

Apple Watch Screen: 
![undefined - Imgur](https://user-images.githubusercontent.com/110037874/207568751-5223ba5d-c833-4acd-a386-32f146e2ff30.jpg)

Postman Screen: 
![undefined - Imgur](https://user-images.githubusercontent.com/110037874/207568188-bdf796da-e97b-4aff-b45d-07a0f4c1fbfc.png)

Version:
![Screenshot 2022-12-14 at 3 48 48 PM](https://user-images.githubusercontent.com/110037874/207569164-85653a7e-bbeb-4544-83d5-9d4d8ba90172.png)

Follow up thread: https://stackoverflow.com/questions/74637344/problem-using-fcm-in-an-independent-watchos-app

### Reproducing the issue

- Create a new WatchOS app.
- Create a new iOS app on Firebase and put the bundle identifier of the watchOS target
- Add APN certificate to Cloud messaging configuration in Firebase Settings
- Add Capability for background fetch and push notification in watchOS target
- Add the following to EntryPoint of WatchOS app
```swift
import SwiftUI
import Firebase

@main
struct Watch_Watch_AppApp: App {
    @WKApplicationDelegateAdaptor(FCMWatchAppDelegate.self) var appDelegate
    var body: some Scene {
        WindowGroup {
            AuthenticationView()
        }
    }
}

// MARK: - WKApplicationDelegate
class FCMWatchAppDelegate: NSObject, WKApplicationDelegate, MessagingDelegate {
  func applicationDidFinishLaunching() {
    FirebaseApp.configure()
    let notificationCenter = UNUserNotificationCenter.current()
    notificationCenter.requestAuthorization(options: [.alert, .sound]) { granted, error in
      if granted {
        WKApplication.shared().registerForRemoteNotifications()
      }
    }
    Messaging.messaging().delegate = self
  }

  func didRegisterForRemoteNotifications(withDeviceToken deviceToken: Data) {
    // Method swizzling should be disabled in Firebase Messaging on watchOS.
    // Set the APNS token manually as is done here.
    // More information on how to disable -
    // https://firebase.google.com/docs/cloud-messaging/ios/client#method_swizzling_in
    print("APNS didRegisterForRemoteNotifications. Got device token \(deviceToken)")
    Messaging.messaging().apnsToken = deviceToken
  }
}

// MARK: - FCM MessagingDelegate
extension FCMWatchAppDelegate {
  func messaging(_ messaging: Messaging, didReceiveRegistrationToken fcmToken: String?) {
    // Use this FCM token to test sending a push using API or Firebase Console
    print("FCM - didReceiveRegistrationToken \(String(describing: fcmToken))")
  }
}
```
- Run App on real device and get FCM token from Xcode debug console
- Use the FCM token to Start a FCM campaign on CloudMessaging of Firebase and try to send a test notification
- Use Postman to send a push notification to the apple watch using the token printed in Xcode debug console

### Firebase SDK Version

9.0.0

### Xcode Version

Version 14.1 (14B47b)

### Installation Method

Swift Package Manager

### Firebase Product(s)

Messaging

### Targeted Platforms

watchOS

### If using Swift Package Manager, the project's Package.resolved

<details>
<summary>Expand <code>Package.resolved</code> snippet</summary>
<br>

```json

Replace this line with the contents of your Package.resolved.

```

</details>

### If using CocoaPods, the project's Podfile.lock

<details>
<summary>Expand <code>Podfile.lock</code> snippet</summary>
<br>

```yml

Replace this line with the contents of your Podfile.lock!

```

</details>

**Comment by aashishpatil-g** [comment]
@sayan-futurehook 

Per the issue description, the Firebase SDK being used is 9.0.0. 

Can you try this on Firebase SDK 10.1.0 or later? We fixed an issue related to Xcode 14 and the way Standalone Watch Apps are treated on Xcode 14. 

You might be running into this - https://github.com/firebase/firebase-ios-sdk/issues/10147, 
which was fixed in 10.1.0

**Comment by sayan-futurehook** [comment]
After updating when I am pushing notifications, it is working. Thank you so much ❤️
